frontend/src/pages/CourseCreator/CourseCreateor.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+
+import EditorTabs from 'components/EditorTabs';
+import { ButtonLabels } from 'constants/ButtonLabels';
+import { PATHS } from 'constants/routes';
+import PageTitle from 'components/PageTitle';
+import transformRoute from 'utils/helpers/paths/transformRoute';
+
+import DefinitionStep from 'pages/CourseEditor/DefinitionStep';
+import SkillsStep from 'pages/CourseEditor/SkillsStep';
+import LessonsStep from 'pages/CourseEditor/LessonsStep';
+import TestStep from 'pages/CourseEditor/TestStep';
+import { BackButton, InnerWrapper } from 'pages/CourseEditor/styled';
+
+import { ICourseCreatorProps } from './types';
+
+const CourseCreator: FC<ICourseCreatorProps> = ({ isCreateCourseMode, ...props }) => (
+  <PageTitle title="Create Course">
+    <InnerWrapper>
+      <BackButton
+        variant="medium"
+        color="primary"
+        component={Link}
+        to={transformRoute(PATHS.coursesList)}
```

Comment:
do we really need transformRoute here?
```suggestion
        to={PATHS.coursesList}
```